src/legacy/core_plugins/vis_type_vega/public/_vega_editor.scss

```diff
@@ -1,41 +1,5 @@
-.visEditor--vega {
-  .visEditorSidebar__config {
-    padding: 0;
-    // Makes sure the vega options dropdown menu is visible
-    overflow: inherit;
-  }
-
-  // The following is necessary for the Vega editor to expand to full height of the editor panel
-  .visEditorSidebar__config,
-  .visEditorSidebar__options {
-    @include flex-parent(1, 1, auto);
-
-    > * {
-      @include flex-parent(1, 1, auto);
-    }
-  }
-
-  @include euiBreakpoint('xs', 's', 'm') {
-    .visEditor__collapsibleSidebar {
-      flex-grow: 1;
-    }
-  }
-}
-
-
 .vgaEditor {
-  @include flex-parent(1, 1, auto);
   position: relative;
-
-  @include euiBreakpoint('xs', 's', 'm') {
-    min-height: $euiSize * 15;
```

Comment:
Please also double check small screens. It used to split vertically with internal scroll now the editor always displays it's full height instead of scrolling

<img src="https://d.pr/free/i/joj68D+" width="50%" />

Reply:
Done